tests/integration/goldens/eventarc/google/cloud/eventarc_v1/services/eventarc/client.py

```diff
@@ -545,6 +547,26 @@ def _validate_universe_domain(self):
             EventarcClient._compare_universes(self.universe_domain, self.transport._credentials))
         return self._is_universe_domain_valid
 
+    def _add_cred_info_for_auth_errors(
```

Comment:
why do tests have the actual logic for appending error details?

Reply:
These are golden files. The `tests/integration/goldens/` folder contains the client code generated by the generator for 5 selected apis (redis, eventarc, etc)